refinery/core/models.py

```diff
@@ -214,6 +214,11 @@ def duplicate_slug_check(self):
     def clean(self):
         # Check if model being saved/altered in Django Admin has a slug
         # duplicated elsewhere.
+
+        # Removes trailing/leading white spaces
```

Comment:
The duplicated logic here is necessary for the special case where a slug is set to just whitespace or None (which we want to allow duplicates of) There may be a better way to do this, but we need the assignment in the first `if self.slug`  so that the second if block isn't reached if a slug is `None` or `""`